CHANGELOG.md

```diff
@@ -127,6 +145,8 @@ The following are deprecated and will be removed in a future release. For anythi
 - Attribution.iconClass deprecated since 4.27. Use icon instead.
 - BasemapGallery.iconClass deprecated since 4.27. Use icon instead.
 - BasemapLayerList.iconClass deprecated since 4.27. Use icon instead.
+- BasemapLayerList.multipleSelectionEnabled depricated since 4.29. Use selectionMode instead.
+- BasemapLayerList.editingEnabled depricated since 4.29. Use dragEnabled, selectionMode, and visibleElements.editButton instead.
```

Comment:
```suggestion
- BasemapLayerList.multipleSelectionEnabled deprecated since 4.29. Use selectionMode instead.
- BasemapLayerList.editingEnabled deprecated since 4.29. Use dragEnabled, selectionMode, and visibleElements.editButton instead.
```